FindOnPage.md

```diff
@@ -0,0 +1,664 @@
+# WebView2Find API
+
+## Background
+
+The WebView2Find API offers methods and events for text finding and navigation
+within a WebView2 control. It enables developers to programmatically initiate find
+operations, navigate find results, suppress default UI, and customize find configurations
+like query and find direction. It also tracks the status of operations, indicating
+completion, match count changes, and match index changes.
+
+## Examples
+
+
+#### Description
+
+To initiate a find operation in a WebView2 control, use the `StartFind` method.
+This method allows setting the find term and find parameters via the
+`ICoreWebView2FindConfiguration` interface. Only one find session can be active per
+webview environment. Starting another with the same configuration will adjust
+the active match index based on the selected Find Direction.
+### Create/Specify a Find Configuration
+#### WIN32 C++
+
+```cpp
+
+//! [InitializeFindConfiguration]
+wil::com_ptr<ICoreWebView2FindConfiguration> AppWindow::InitializeFindConfiguration(const std::wstring& findTerm)
+{
+    // Query for the ICoreWebView2Environment5 interface.
+    auto webView2Environment5 = m_webViewEnvironment.try_query<ICoreWebView2Environment5>();
+    CHECK_FEATURE_RETURN(webView2Environment5);
+
+    // Initialize find configuration/settings
+    wil::com_ptr<ICoreWebView2FindConfiguration> findConfiguration;
+    CHECK_FAILURE(webView2Environment5->CreateFindConfiguration(&findConfiguration));
+    CHECK_FAILURE(findConfiguration->put_FindTerm(findTerm.c_str()));
+    CHECK_FAILURE(findConfiguration->put_IsCaseSensitive(false));
+    CHECK_FAILURE(findConfiguration->put_ShouldMatchWord(false));
+    CHECK_FAILURE(findConfiguration->put_FindDirection(COREWEBVIEW2_FIND_DIRECTION_FORWARD));
```

Comment:
I don't know what the default find UI looks like. Does it have UI to control these options? E.g. is there a TextBox for the find term and checkboxes for case-sensitive/match-word? 
If so, what does interacting with this UI do?
Similarly, is there a 'next' button? If so, what does clicking on it do?

In general, it would be good to describe what using this API with the default find UI looks like. I can understand how using this API would work in the case where I am suppressing the default UI. But in the case where I am relying on the default UI, it's unclear what I would do with this API. What is the scenario that is trying to be achieved?

Reply:
See above to add screenshot and more details on the scenario.